VMware/SampleAdminScenarios/vSan VMDK Migration/FlatVmdkConverter.psm1

```diff
@@ -0,0 +1,547 @@
+Add-PSSnapin Citrix* -ErrorAction Stop
+Import-Module .\vSphereConnect.psm1
+Import-Module .\vSphereHelper.psm1
+        
+        function ConvertVmdkFiles() {
+            <#
+            .SYNOPSIS
+            Converts the FLAT format disk to stream optimized format
+            .DESCRIPTION
+             This powershell script  converts the FLAT format disk to stream optimized format for provisioned VM or VM's in a provisioning scheme
+            .EXAMPLE
+             ConvertVmdkFiles -MCSCatalogName Test -CloudCustomerId 123 -CloudCustomerApiKey 123  -vCenterServerAddress 123 
+             ConvertVmdkFiles -VMName Test -CloudCustomerId 123 -CloudCustomerApiKey 123  -vCenterServerAddress 123 
+            .PARAMETER ProvisioningSchemeName
+                MCS Provisioning Scheme name
+            .PARAMETER VMName
+             The name of a virtual machine
+            .PARAMETER CloudCustomerId
+             The customer id
+            .PARAMETER CloudCustomerApiKey
+             The customer api key
+            .PARAMETER vCenterServerAddress
+             The vCenter server address
+            .PARAMETER ForceRemoveFlatIdentity
+             The flag to remove flat identity
+            .NOTES
+             Version      : 1.0.0
+             Author       : Cloud Software Group, Inc.
+            
+        #>
```

Comment:
please move this comment and copyright below line to the top of the file.
Here is a reference
https://github.com/citrix/citrix-mcs-sdk-samples/blob/main/VMware/SampleAdminScenarios/Add%20Machine%20Catalog/Add-MachineCatalog.ps1

Reply:
please do this in the other files